### DISCUSSION TITLE
preview in rider of cross platform template - browser doesn't work

### DISCUSSION THREAD

**Discussion by LukAss741** [AvaloniaUI/Avalonia#17242] [Q&A] [answered]
### Describe the bug

My preview of cross  platform application - browser in Rider doesn't work.

### To Reproduce

Install AvaloniaRider plugin 1.5.1 to rider.
Create a new project with avalonia cross platform template.
Build
Click preview only

Results in infinite loading circle and this in the log:
```
"C:\Program Files\dotnet\dotnet.exe" exec --runtimeconfig C:\Users\lukas\RiderProjects\MMbot6\MMbot.Browser\bin\Debug\net8.0-browser\MMbot.Browser.runtimeconfig.json --depsfile C:\Users\lukas\RiderProjects\MMbot6\MMbot.Browser\bin\Debug\net8.0-browser\MMbot.Browser.deps.json C:\Users\lukas\.nuget\packages\avalonia\11.1.0\buildTransitive\..\tools\netstandard2.0\designer\Avalonia.Designer.HostApp.dll --transport tcp-bson://127.0.0.1:50279/ --method avalonia-remote C:\Users\lukas\RiderProjects\MMbot6\MMbot.Browser\bin\Debug\net8.0-browser\MMbot.Browser.dll
The specified deps.json [C:\Users\lukas\RiderProjects\MMbot6\MMbot.Browser\bin\Debug\net8.0-browser\MMbot.Browser.deps.json] does not exist
Process terminated with exit code -2,147,450,751
```

### Avalonia version

11.1.0

### OS

Windows

**Comment by MrJul** [comment, marked as answer]
The previewer needs a desktop project to work.
Just switch the previewer project to the Desktop one:
![image](https://github.com/user-attachments/assets/fa82305f-57e9-4c2f-b44a-5ddb1f9db0b9)

AvaloniaRider hides the mobile projects from the previewer by default, but doesn't seem to filter out the browser project like AvaloniaVS does.
